### ISSUE TITLE
How to identify  same person in multicamera using cosine distance metric.

### ISSUE THREAD

**Issue by Basavarajdocketrun** [layumi/Person_reID_baseline_pytorch#201]
I am very beginer to this field. what i have understood is that features extracted from person image are compared using cosine distance metric and ranking is done based on the distance.But if i have same person in the another camera how can i recognise him using same cosine distance metric or i need to use other metric loss such as triplet loss to recognize him. I want to track same person in multiple camera with same ID. pls help

**Comment by layumi** [comment]
Hi @Basavarajdocketrun 
When evaluation, you could use the cosine distance as well.  Just extract the new feature of the same person in the new camera and do the comparison.

By the way, either triplet loss or softmax loss is only for the training stage. We do not calculate the loss as the inference time.

**Comment by Basavarajdocketrun** [comment]
thanks @layumi  for answering. And do i need to set any threshold for the comparison value so that if the person is not present in the new camera it should not associate the tracking ID to the other closest one.  If yes what should be the optimum value for the threshold
